spritzle/tests/test_core.py

```diff
@@ -20,22 +20,20 @@
 #   Boston, MA    02110-1301, USA.
 #
 
-from unittest.mock import patch, MagicMock
-import tempfile
-import os
 
-import spritzle.core
+async def test_save_session_state(core):
+    state_file = core.state_dir / 'session.state'
+    await core.start()
+    assert not state_file.is_file()
+    await core.save_session_state()
+    assert state_file.is_file()
```

Comment:
Maybe also test that you can properly bdecode the file and that it has greater than 0 keys.

Reply:
Yep, good plan, done.